index.js

```diff
@@ -218,6 +218,14 @@ module.exports = function (buf) {
 		};
 	}
 
+	// needs to be before `ogg` check
+	if (buf[28] === 0x4F && buf[29] === 0x70 && buf[30] === 0x75 && buf[31] === 0x73 && buf[32] === 0x48 && buf[33] === 0x65 && buf[34] === 0x61 && buf[35] === 0x64) {
+		return {
+			ext: 'opus',
+			mime: 'audio/ogg; codecs=opus'
```

Comment:
You're right, `audio/opus` is clearer and makes more sense. All the 3 first links given above mention the other mime, which is kind of misleading... Anyway, I just updated that MIME.